src/Microsoft.Performance.Toolkit.Engine.Tests/TestCookers/Source123/Source123DataSource.cs

```diff
@@ -44,6 +51,8 @@ protected override bool IsDataSourceSupportedCore(IDataSource dataSource)
                    Path.GetExtension(dataSource.Uri.LocalPath));
         }
 
+        public override IEnumerable<Option> CommandLineOptions => options;
```

Comment:
Now that we don't directly use these PSRs in the unit tests, can all these be removed?

They could be used for integration testing the resolver if we want to do that, though.

Reply:
Nevermind, didn't see the engine create info tests 😄